doc/nvim-tree-lua.txt

```diff
@@ -1889,6 +1894,12 @@ node.navigate.git.prev_skip_gitignored()
 node.navigate.diagnostics.next()
     Navigate to the next item showing diagnostic status.
 
+                              *nvim-tree-api.node.navigate.diagnostics.next_recursive()*
+node.navigate.diagnostics.next_recursive()
+    Same as |node.navigate.diagnostics.next()|, but navigates to next item
+    showing diagnostic status recursively. Needs `diagnostics.show_on_dirs`
+    and `diagnostics.show_on_open_dirs` set to `true`.
```

Comment:
Thank you - that is very clear.

Reply:
I'm wondering now: my latest tests showed success for diagnostics when `diagnostics.show_on_dirs = true` and `diagnostics.show_on_open_dirs = false`… what about you?

Reply:
Yes... that's the most useful use case IMO